static/app/utils/metrics/mri.spec.tsx

```diff
@@ -84,6 +84,7 @@ describe('parseMRI', () => {
     ['d:transactions/duration@millisecond', 'transaction.duration'],
     ['d:spans/duration@millisecond', 'span.duration'],
     ['d:spans/exclusive_time@millisecond', 'span.self_time'],
+    ['g:spans/self_time@millisecond', 'span.self_time'],
```

Comment:
WiIl we get duplicated metric entries somewhere? 🤔

Reply:
Yes, but i don't know how else to call it 😂

Reply:
Hmm, seems sub-optimal to me...

Reply:
moved to `self_time` and `exclusive time`